connectors/csws/connector/src/main/java/org/apache/manifoldcf/csws/CswsSession.java

```diff
@@ -535,14 +543,18 @@ public PageHandle getAllUsers()
     throws ManifoldCFException, ServiceInterruption {
     try {
       final SingleSearchRequest singleSrchReq = new SingleSearchRequest();
-      singleSrchReq.setDataCollectionSpec("'LES Enterprise'");//Livelink Enterprise Server
+      // TODO: Apparently, this is different in some implementations. Fetch dynamically or make an input field.
+      singleSrchReq.setDataCollectionSpec("'LES Athene'"); //Livelink Enterprise Server
       singleSrchReq.setQueryLanguage("Livelink Search API V1"); //Search Query Language API
       singleSrchReq.setFirstResultToRetrieve(start + 1);
       singleSrchReq.setNumResultsToRetrieve(count);
       if (orderingColumn != null) {
         singleSrchReq.setResultOrderSpec("sortByRegion="+orderingColumn+"&sortDirection=ascending");
       }
-      singleSrchReq.setResultSetSpec("where1=(\"OTParentID\":"+parentID+" AND ("+searchSpec+")");
+
+      // TODO: Don't ignore the searchSpec. This is part of the NodeRights issue.
+      //singleSrchReq.setResultSetSpec("where1=(\"OTParentID\":"+parentID+" AND ("+searchSpec+"))");
+      singleSrchReq.setResultSetSpec("where1=(\"OTParentID\":"+parentID+")");
```

Comment:
Overlooked this. The searchSpec seems not to work for db systel's csws system. But i am worried, that removing filtering completely has also unwanted side effects

Reply:
There is a Patch in Trunk now, which fixes the connector to properly handle the searchSpec. No need to skip it any more...